### ISSUE TITLE
Validation and Test set for DeepFashion In-shop

### ISSUE THREAD

**Issue by hugoych** [salesforce/hierarchicalContrastiveLearning#5]
Hi, 

I couldn't find the info in the paper or the repo. What is your validation set and test set?
I'm not familiar with this dataset and I see three sets in DeepFashion: "train", "gallery" and "query". I wonder which one was which?

Thanks a lot!
